Source/WebCore/workers/WorkerMessagingProxy.cpp

```diff
@@ -331,6 +331,17 @@ void WorkerMessagingProxy::postExceptionToWorkerObject(const String& errorMessag
     });
 }
 
+void WorkerMessagingProxy::reportErrorToWorkerObject(const String& errorMessage)
+{
+    if (!m_scriptExecutionContext)
+        return;
+
+    m_scriptExecutionContext->postTask([this,  errorMessage =  errorMessage.isolatedCopy()] (auto&) {
+        if (auto* workerObject = this->workerObject())
```

Comment:
`auto*` -> `RefPtr`.

Please be careful in new code to not introduce new raw pointers as we are moving towards using smart pointers everywhere.

Reply:
Done